test/image_test.py

```diff
@@ -750,6 +750,48 @@ def f(self):
         print("bar!", VARIABLE_6)
 
 
+class FooInstance:
+    bar: str = "5"
+    barbar: str = "2"
```

Comment:
Slight suggestion to make the tests below easier to follow, could name the attributes something like `used_by_build_method`, `not_used_by_build_method`

Reply:
Done